app/components/PizzaMeetingButton/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from '@webkom/lego-bricks';
+import { ContentList } from 'app/components/ContententList';
+import Tooltip from 'app/components/Tooltip';
+import { MeetingInvitationStatus } from 'app/store/models/MeetingInvitation';
+import type { EntityId } from '@reduxjs/toolkit';
+import type { MeetingInvitationWithUser } from 'app/reducers/meetingInvitations';
+import type { ReactionsGrouped } from 'app/store/models/Reaction';
+
+type Props = {
+  meeting: {
+    reactionsGrouped?: ReactionsGrouped[];
+    invitations: EntityId[];
+    reactions?: { author: { fullName: string }; emoji: string }[];
+  };
```

Comment:
Please just use the actual meeting model from `app/store/models/Meeting.d.ts`

Reply:
I get an err that reactions does not exist on meeting. Is it possible to use Meeting interface and add reaction as a field?

Reply:
hmm is it even supposed to have a `reactions` field? Is the `reactionsGrouped` not sufficient?

Reply:
I'm not sure. I have copied a lot from `app/components/LegoReactions`, and they do it that way. I am not sure if I have understood this right but it also looks like the user field in `reactionsGrouped` is empty.

Reply:
@eikhr should verify this since he made the ts model and reducer

Reply:
Ah, yes we need to use reactions and not `reactionsGrouped` to get the list of users who reacted. I didn't review that PR so not sure exactly how it is implemented but that's why `reactions` exists.